var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -18,10 +18,10 @@ class Boost(Package):
        across a broad spectrum of applications. The Boost license
        encourages both commercial and non-commercial use.
     """
-    homepage = "http://www.boost.org"
-    url      = "http://downloads.sourceforge.net/project/boost/boost/1.55.0/boost_1_55_0.tar.bz2"
```

Comment:
@hainest I seem to recall some issue that made us switch from https to http for boost, is that correct?

Reply:
yeah looks like it redirects to the sourceforge page - I'll return back to http.

Reply:
This should be fixed.

Reply:
Not sure if this also applied to the list_url or not.

Reply:
Maybe - I just changed it back.